lib/main.dart

```diff
@@ -32,26 +39,44 @@ List<String> checkboxCollectionNames = [
   'PersonalPlan-Distractions',
   // Add the new table name
 ];
-Future<FirebaseApp> initializeApp() async {
+TimeOfDay calculateTime(int h, int m) {
```

Comment:
why do we have it here? isn't this a duplication with the same function in `notification_service` ?
maybe we need `dateTimeUtil` object?

Reply:
I don't think we need an object for it, I just moved this function into the NotificationsService